pages/install/smelt.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: Moto 360 2015
 deviceName: smelt
+repo: carp/smelt
 section: install
-layout: aw-install
+layout: aw-install-smelt
```

Comment:
Instead of adding a new layout, add a new variable with a default value for the windows adb link and we can overwrite it for this watch